BE/LegalEntities/FormalBusinessOrganizations.rdf

```diff
@@ -209,12 +213,63 @@
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:label>organizational sub-unit</rdfs:label>
-		<skos:definition>any department, service, or other entity within a larger organization that only has full recognition within the context of that organization, but requires identification for some purpose</skos:definition>
+		<skos:definition>any department, service, and other entity within a larger organization that only has full recognition within the context of that organization, but requires identification for some purpose</skos:definition>
 		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">https://www.iso.org/obp/ui/#iso:std:iso-iec:6523:-1:ed-1:v1:en</fibo-fnd-utl-av:adaptedFrom>
 		<fibo-fnd-utl-av:explanatoryNote>In other words, it is not a legal entity in its own right.</fibo-fnd-utl-av:explanatoryNote>
 		<fibo-fnd-utl-av:synonym>organization part</fibo-fnd-utl-av:synonym>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-be-le-fbo;TaxIdentificationScheme">
```

Comment:
This does not belong in FBO since it applies just as much to people. It's a Government thing surely.

Reply:
I've moved the definition of the tax id and scheme classes to FND, but retained the union in FBO, because organization identifier is defined in FBO, and a tax id is either a national identification number for a person or an organization identifier.